provider/datagen/src/lib.rs

```diff
@@ -540,7 +540,7 @@ fn test_keys_from_file() {
 #[test]
 fn test_keys_from_bin() {
     // File obtained by changing work_log.rs to use `try_new_with_buffer_provider` & `icu_testdata::small_buffer`
-    // and running `cargo +nightly-2022-04-18 wasm-build-release --examples -p icu_datetime --features serde \
+    // and running `cargo +nightly-2023-01-01 wasm-build-release --examples -p icu_datetime --features serde \
```

Comment:
Did you actually regen this without a diff?

Reply:
Oh, no I haven't.